Assignment.md

```diff
@@ -9,34 +9,70 @@
 1. Write a `for` loop that will iterate from `0` until `10`.
 1. For each iteration of the `for` loop, it will multiply the number by `9` and log the result (e.g. `"2 * 9 = 18"`).
 
+console.log("starting loop ...");
 **Bonus:**
 
 - Use a nested for loop to show the tables for every multiplier from 1 to 10 (100 results total)
 
-```js
+````js
 // Your Answer
-```
+`console.log("starting loop ...");
+
+for (let i = 0; i < 11; i++) {
+console.log(i * 9);
+}
+console.log("Ending loop ...");``
+
+
 
 ### 2: The Grade Assigner
 
 Check the results for every value from 60 to 100 - so your log should show "For 89, you got a B. For 90, you got an A.", etc.
 
 ```js
 // Your Answer
-```
+for (let i = 60; i < 101; i++) {
+  console.log(i);
+
+  if (i === 89) {
+    console.log("you got a B");
+  }
+  if (i === 90) {
+    console.log("you got an A. ");
+  }
+}```
 
 ### 3: Favorite TV Shows
 
 1. Create an Array to hold your favorite TV shows
 1. For each show, print to the console a string like:
 
-   ```
+   <!-- ```
    My #1 choice is Person of Interest
    My #2 choice is Battlestar Galactica
-   ```
-
-```js
-// Your Answer
+````
+
+```js -->
+//Your Answer
+const favoriteTvShows = [
+  "Desmonds",
+  "BBC New",
+  "Telekoma",
+  "Giving Time",
+  "Akuna Matata",
+  "TINSEL"
+];
+
+for (let i = 0; i < favoriteTvShows.length; i++) {
+  console.log(favoriteTvShows[i]);
+
+  if (i === 0) {
```

Comment:
Good job on attempting this problem, you did well. Though, you went a bit overboard. You did not need to use any conditions and show only 2 of your favorite shows. The prompt was asking you to list each of your favorite shows along with their standing in your favorites.

That would have been simpler as well.